tox.ini

```diff
@@ -34,54 +38,54 @@ whitelist_externals =
 description = Runs all non-Python3-based server unit/functional tests
 passenv = PBENCH_UNITTEST_SERVER_MODE
 deps =
-   -r{toxinidir}/server/requirements.txt
-   -r{toxinidir}/server/test-requirements.txt
+    -r{toxinidir}/server/requirements.txt
```

Comment:
is the extra space actually necessary?

Reply:
It is necessary for consistency.  Having 3 space indentation in some places, and 4 space indentation in other places, can lead to unnecessary changes as the file evolves over time and people "pick" the indentation.  This is why we are using `black` in other parts of the tree.

It is also why I place the indentation fixes in a separate commit here.

Reply:
FWIW, my previous comment that was supposed to be posted was not accepted by GitHub for some reason.